docs/installation/server-config/index.rst

```diff
@@ -211,15 +209,14 @@ The default number of replicas for the topics created by KSQL. The default is on
 listeners
 ^^^^^^^^^
 
-The maximum number of memory bytes to be used for buffering across all threads. The default value in KSQL is ``10000000`` (~ 10 MB).
-Here is an example to change the value to ``20000000`` by using the KSQL command line:
+Set the port for the KSQL Server to listen on. This defaults to http://localhost:8088. To listen for requests from all
+servers on port 80, updated it accordingly in your ``ksql-server.properties`` file.
```

Comment:
> To listen for requests from all servers on port 80...

I'm not sure I understand what the 'from all servers' part is meaning.  Could this not simple be:

> To listen for requests on port 80.

Also, I think `updated` should be `update`.

Reply:
+1 for @big-andy-coates 's suggestion.